.github/CONTRIBUTING.md

```diff
@@ -186,32 +186,31 @@ At any rate, let's have some fun!
 ### Articles
 
 As a bonus to this repository, there are associated articles for every script.
-However, the articles do take awhile to write, so you can help by writing them
+However, the articles do take a while to write, so you can help by writing them
```

Comment:
I tried looking into this one, and I have no idea which version is correct. haha

I'll take your word for it.

Source: https://www.merriam-webster.com/words-at-play/awhile-usage

Reply:
articles do take a while to write is the proper one.